app/presentation/api.py

```diff
@@ -14,13 +14,25 @@ class OrganizationArgs(BaseModel):
     q: Optional[str] = None
     page: Optional[int] = 1
     page_size: Optional[int] = 20
+    sort: Optional[str] = None
     badge: Optional[str] = None
 
+    @validator('sort')
+    def sort_validate(cls, value):
+        sorts = [
+            'title', 'created', 'last_modified', 'reuses', 'datasets', 'followers', 'views'
```

Comment:
Seems like sorting on text field `title` is not supported out-of-the-box. Maybe we can remove this sort support (on udata as well)?